pkg/quota/v1/evaluator/core/persistent_volume_claims.go

```diff
@@ -64,6 +64,24 @@ func V1ResourceByStorageClass(storageClass string, resourceName corev1.ResourceN
 	return corev1.ResourceName(string(storageClass + storageClassSuffix + string(resourceName)))
 }
 
+// pvcVACResources are the set of static resources managed by quota associated with pvcs.
+// for each resource in this list, it may be refined dynamically based on volume attributes class.
+var pvcVACResources = []corev1.ResourceName{
+	corev1.ResourcePersistentVolumeClaims,
+}
+
+// volumeAttributesClassSuffix is the suffix to the qualified portion of volume attributes class resource name.
+// For example, if you want to quota storage by a volume attributes class, you would have a declaration
+// that follows <volume-attributes-class>.volumeattributesclass.storage.k8s.io/<resource>.
+// For example:
+// * bronze.volumeattributesclass.storage.k8s.io/persistentvolumeclaims: "5"
```

Comment:
What about requests.storage?

Reply:
according to https://github.com/kubernetes/enhancements/tree/master/keps/sig-storage/3751-volume-attributes-class#administrator-quota-restrictions, it is not mentioned to support requests.storage.

/cc @sunnylovestiramisu

for confirmation.

Reply:
It is not supported.